bin/setup

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require "pathname"
+require "fileutils"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"]
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+APP_ROOT = Pathname.new File.expand_path("../../",  __FILE__)
 
-Dir.chdir APP_ROOT do
+chdir APP_ROOT do
   # This script is a starting point to setup your application.
   # Add necessary setup steps to this file:
 
-  puts "== Installing dependencies =="
-  system "gem install bundler --conservative"
-  system "bundle check || bundle install"
+  puts "== Building Dashboard App =="
+  puts "RAILS_ENV               = #{ENV["RAILS_ENV"]}"
+  puts "RAILS_RELATIVE_URL_ROOT = #{ENV["RAILS_RELATIVE_URL_ROOT"]}"
```

Comment:
Display a different message if `RAILS_RELATIVE_URL_ROOT` not set? Something like:

>RAILS_RELATIVE_URL_ROOT not specified; rake tasks will use whatever is defined in dotenv files (if using dotenv) or framework default

Or just omit if not set?